[IMP] accounting: improve clarity of internal transfers page (#17300)
odoo/documentation: content/applications/finance/accounting/bank/internal_transfers.rst

Reviewer: ```suggestion
An internal transfer account is automatically created in your database based on your company's
```
I know you didn't modify this, but since we are there ;)

modify the default :guilabel:`Internal Transfer` account, go to :menuselection:`Accounting -->
Configuration --> Settings` and scroll down to the :guilabel:`Default Accounts` section.

While it is possible to manually :ref:`write off <accounting/reconciliation/set-account>` internal

> transfers to this internal transfers account, it is recommended to instead use the the

Reviewer 2: ```suggestion
transfers to this internal transfers account, it is recommended to instead use the
```

:guilabel:`Internal Transfers` :doc:`reconciliation model <reconciliation_models>`. If, for whatever

> reason, the :guilabel:`Internal Transfers` reconciliation model does not exist, create it manually:

Reviewer: ```suggestion
:guilabel:`Internal Transfers` :doc:`reconciliation model <reconciliation_models>`. If
the :guilabel:`Internal Transfers` reconciliation model does not exist, create it manually:
```
"_for whatever reason_" I would remove that, as it doesn't give useful information.

#. :ref:`Create a new reconciliation model <accounting/rec-models/config>`, and fill in the
   :guilabel:`Name` field (e.g., `Internal Transfers`).
#. Leave the default values for the :ref:`matching conditions <accounting/rec-models/conditions>`

>    and leave the model set to :guilabel:`Manual` to ensure the model is available to be selected for

Reviewer: Maybe it's just me, but "leave the default values" could be slightly confusing, as no value is selected. Maybe saying "_do not select specific matching conditions_"? It's still understandable the way it is, but it could be slightly clearer.

(assuming that's the config we are looking for here)
<img width="289" height="226" alt="Image" src="https://github.com/user-attachments/assets/530c6c99-221d-4f16-b690-27e256e39419" />

Reviewer 2: @xpl-odoo , @dade-odoo had originally put something along the lines of 'leave the matching conditions blank', which I also thought could be confusing. Your suggestion could work well, IMO.

   all transactions.
#. In the :guilabel:`Counterpart Items` tab, click :guilabel:`Add a line`.
#. Set the :guilabel:`Account` to the internal transfer account (the specific account name and code

>    vary based on the localization).

Reviewer: ```suggestion
   vary by localization).
```

Register an internal transfer from one bank to another
======================================================

When money is transferred from one bank or cash account to another, that amount appears as two
transactions on the corresponding journals, whether the transactions are created manually, via
import, or via :doc:`bank synchronization <bank_synchronization>`. When :ref:`reconciling the
transaction <accounting/reconciliation/reconcile>`, select the :guilabel:`Internal Transfers`
:doc:`reconciliation model <reconciliation_models>` button. This action writes the transaction off
to the :guilabel:`Internal Transfer` account.

.. important::
   Remember to reconcile both the outgoing transaction (on the journal that sends the payment) and
   the incoming transaction (on the journal that receives the payment) with the internal transfers
   account using the :guilabel:`Internal Transfers` reconciliation model button.

Once each bank journal's transaction is reconciled using the :guilabel:`Internal Transfers`
reconciliation model, the internal transfer account is balanced, and the balances of both bank
accounts are updated to reflect the transfer.

.. example::

>    When transferring $1000 from Bank Account A to Bank Account B, each of the two bank journals have

Reviewer: ```suggestion
   When transferring $1,000 from Bank Account A to Bank Account B, each of the two bank journals has
```
(the comma is for consistency only)

modify the default :guilabel:`Internal Transfer` account, go to :menuselection:`Accounting -->
Configuration --> Settings` and scroll down to the :guilabel:`Default Accounts` section.

If, for whatever reason, the :guilabel:`Internal Transfers` reconciliation model button does not

> exist, create it manually:

Reviewer 2: I'm wondering if this would benefit from a little more context (a bit like you give for the internal transfer account)? Probably doesn't help that I'm totally new to this, but ... I expected this to be a continuation of the previous paragraph, with some instructions 🙈 

Also, I guess it's more that the Internal Transfers reconciliation _model_ doesn't exist, rather than just the 'button' (that's not a button ... or maybe it is sometimes a button?) not showing. Because they do just then need to create the actual model manually.

Author: Both good points! Pushing my changes in just a few, let me know what you think :)

Reviewer 2: This works really well!

> #. :ref:`Create a new reconciliation model <accounting/rec-models/config>`, and fill in the
>    :guilabel:`Name` field (e.g., `Internal Transfers`).

Reviewer 2: ```suggestion
#. :ref:`Create a new reconciliation model <accounting/rec-models/config>`, fill in the
   :guilabel:`Name` field (e.g., `Internal Transfers`), and leave the default values for the matching conditions.
```

Is there a reason you only mention adding a name, and nothing about the matching conditions, even maybe just to say not to add any? (It's probably only becase I clicked the link to the 'reconciliation models' page, saw how to create a new one, added the name ... and then was already looking at the  step about counterpart itmes ... then wondering about the rest of the first part.)

Author: You're right, it's because you don't need to set any matching conditions. I'll mention to leave them blank and to leave the model as "manual" instead of "automated" :) Let me know what you think

Reviewer 2: Would it be better to talk about leaving the default values rather than saying leave them blank?

#. In the :guilabel:`Counterpart Items` tab, click :guilabel:`Add a line`.
#. Set the :guilabel:`Account` to the internal transfer account (the specific account name and code

>    vary based on the localization).

Reviewer 2: I guess the Partner field isn't relevant since it's internal, but worth mentioning to be complete?

Author: Personally I don't think it's really worth mentioning, especially because we also have the analytic tags and taxes (hidden by default) fields that we don't mention (and I don't think we need to). If you feel strongly that it should be mentioned, let me know :)

Reviewer 2: Good point. Ok to not mention it then.

#. Leave the :guilabel:`Amount Type` and :guilabel:`Amount` as their default values of
   :guilabel:`Percentage of balance` and :guilabel:`100`.
#. Add a :guilabel:`Label` to identify transactions that are matched using this reconciliation model

>    (i.e., `Internal Transfers`).

Reviewer 2: I guess more an '**e.g.**, `Internal Transfers`' rather than '**i.e.,**' since they're free to choose (even if the goal is to identify internal transfers).

Author: I didn't study my latin :grimacing:  :nerd_face:

> Transfer` account.

Reviewer 2: Add a ref to applications/finance/accounting/bank/reconciliation.rst on 'When **reconciling** the transaction, ...'?

> Transfer` account.

Reviewer 2: ```suggestion
button. This action writes the transaction off to the :guilabel:`Internal
Transfer` account.
```

Just to avoid that mouthful again?

.. important::
   Remember to reconcile both the outgoing transaction (on the journal that sends the payment) and
   the incoming transaction (on the journal that receives the payment) with the internal transfers
   account using the :guilabel:`Internal Transfers` reconciliation model button.

Once each bank journal's transaction is reconciled using the :guilabel:`Internal Transfers`
reconciliation model, the internal transfer account is balanced, and the balances of both bank
accounts are updated to reflect the transfer.

.. example::
   When transferring $1000 from Bank Account A to Bank Account B, each of the two bank journals have
   one journal entry:

>    - The outgoing transaction in the Bank A journal:

Reviewer 2: ```suggestion
.. example::
   When transferring $1000 from Bank Account A to Bank Account B, each of the two bank journals have
   one journal entry:
   
   - The outgoing transaction in the Bank A journal:
```

You're just missing the blank link to make the first bullet appear correctly.
